includes/functions.inc.php

```diff
@@ -2738,6 +2740,26 @@ function getAvatar($user_id) {
 	return false;
 }
 
+/**
+ * Update voting permissions
+ *
+ * @param none
+ * @return none
+ */ 
+
+function updateVotingPermissions() {
+    global $connid, $settings, $db_settings;
+    $voting_min_registered_days = $settings['voting_min_registered_days'];
+    $voting_min_threadcount = $settings['voting_min_threadcount'];
+    $voting_min_postcount = $settings['voting_min_postcount'];
+    $voting_min_ownscore = $settings['voting_min_ownscore'];
```

Comment:
Why do you copy the settings into new variables? You only use them in the database queries without any altration. Here you can use the settings variables in itself instead.